bin/ColorScript/.gitignore

```diff
@@ -0,0 +1,4 @@
+.DS_Store
```

Comment:
Does this need its own `.gitignore`?

Reply:
SPM created it when I ran `git init` (following a tutorial), but since we're not creating a separated repo for this anymore, I will remove it.